sotodlib/tod_ops/jumps.py

```diff
@@ -432,34 +427,36 @@ def twopi_jumps(
     if not isinstance(signal, np.ndarray):
         raise TypeError("Signal is not an array")
     if atol is None:
-        atol = nsigma * std_est(signal.astype(float), ds=win_size)
-        np.clip(atol, 1e-8, 1e-2)
+        atol = nsigma * std_est(
+            signal.astype(float), ds=win_size * 10, win_size=win_size
```

Comment:
The downsampling is only for estimating the noise level so the factor of 10 is semi arbitrary since it assumes we have many windows. This line https://github.com/simonsobs/sotodlib/blob/85be33a586fea9d44bb0aedefde6873ccb8c75ec/sotodlib/tod_ops/jumps.py#L54 clips the down sampling if the TOD is short. 
Perhaps its worth thinking about this more carefully. I'll go stare at some old notes on poisson stats and get back to you...